2/contrib/openshift/base-plugins.txt

```diff
@@ -4,12 +4,8 @@ openshift-client:1.0.14
 
 
 # kubernetes plugin - https://wiki.jenkins-ci.org/display/JENKINS/Kubernetes+Plugin
-# 1.6.0 has the support we added for allowing the use of pod yaml for pod template
-# config from the config panel (there was a flavor for this already with the k8s plugin's
-# pipeline DSL, and 1.6.2 has the support for opting in the transfer of proxy related
-# env vars from the master to the client, and 1.7.1 fixes
-# https://issues.jenkins-ci.org/browse/SECURITY-883
-kubernetes:1.7.1
```

Comment:
I can re-add the above comment if you'd like. I just eagerly removed it as I didn't know how best to update it to keep it relevant.

Reply:
hey @grdryn - how about you add the older SECURITY-883 note back in, but you can remove the other stuff

Also, I went back and double checked, and 1.12.0 already has the changes from @adambkaplan we were waiting on :-)

So I'll merge this once you add the 883 comment back in.

thanks

Reply:
@gabemontero I've updated it now, does it look okay?

Reply:
perfect thanks @grdryn !!